lib/src/components/NxNavigationSidebar/NxNavigationSidebarLinks.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React, { FunctionComponent } from 'react';
+import classnames from 'classnames';
+
+import { NxNavigationSidebarContentProps as Props, nxNavigationSidebarContentPropTypes } from './types';
+export { NxNavigationSidebarContentProps as NxNavigationSidebarLinksProps } from './types';
+
+const NxNavigationSidebarLinks: FunctionComponent<Props> = function NxNavigationSidebarLinks(props) {
+  const { className, children } = props;
```

Comment:
Same issue as in the previous file.  Btw, on Improvement Day yesterday I created a HOC to make this pattern super simple.  It's not ready to merge yet though as I didn't create tests or docs for it: https://github.com/sonatype/sonatype-react-shared-components/blob/improvement-day_simple_components/lib/src/util/withClass.tsx

Reply:
> Same issue as in the previous file. Btw, on Improvement Day yesterday I created a HOC to make this pattern super simple. It's not ready to merge yet though as I didn't create tests or docs for it: https://github.com/sonatype/sonatype-react-shared-components/blob/improvement-day_simple_components/lib/src/util/withClass.tsx

I like your `withClass` HOC, but in the meantime is this similar to what you had in mind? 7edb99c246614a4561d9bdac48c7a15f43f07c87